assets/javascripts/initializers/shared-edits-init.js

```diff
@@ -90,7 +90,10 @@ function initWithApi(api) {
     html(attrs) {
       const contents = this._super(...arguments);
 
-      if (!this.currentUser.staff || !contents.children) {
+      if (
+        !(this.currentUser.staff || this.currentUser.trust_level > 3) ||
+        !contents.children
+      ) {
```

Comment:
Not sure if it's possible in this exact case, but in the case of an anonymous user, `this.currentUser` will be null. So, this should be safer:

```javascript
      if (
        !this.currentUser || 
        !(this.currentUser.staff || this.currentUser.trust_level > 3) ||
        !contents.children
      ) {
```

Reply:
I hear you, but I think that in this very specific case it is safe, given it is an admin only widget